### ISSUE TITLE
[macOS] Cursor shifting to left while typing text

### ISSUE THREAD

**Issue by dphov** [zeit/hyper#577]
```
os x 10.11.6
zsh 5.2 (x86_64-apple-darwin15.0.0) + oh_my_zsh with theme 'powerlevel9k'
hyperterm 0.7.1.36
```

![bug](https://cloud.githubusercontent.com/assets/8191815/17455661/99764886-5bd8-11e6-8909-6d86b59b2cf4.gif)

**Comment by MrRio** [comment]
This is related to https://github.com/zeit/hyperterm/pull/535

**Comment by BenoitAverty** [comment]
@dphov I'm curious, could you tell me which fonts you're using and in which mode you're running powerlevel9k? I'm in the process of making my own conf that has to work on all my computers and it's a bit harder than I anticipated.

**Comment by dphov** [comment]
@BenoitAverty Sure! Current font is `DroidSansMonoAwesome 12` and powerlevel9k config is 
`POWERLEVEL9K_MODE='awesome-patched'
 POWERLEVEL9K_SHORTEN_DIR_LENGTH=3
 POWERLEVEL9K_SHORTEN_STRATEGY="truncate_middle"
 POWERLEVEL9K_STATUS_VERBOSE=false
 POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(status os_icon load context dir vcs)
 POWERLEVEL9K_RIGHT_PROMPT_ELEMENTS=(time)
 POWERLEVEL9K_SHOW_CHANGESET=true
 POWERLEVEL9K_CHANGESET_HASH_LENGTH=6`

I'd like to see your config file when you're done with it

**Comment by BenoitAverty** [comment]
@dphov Thanks. My config lives with my other dotfiles, at http://github.com/BenoitAverty/dotfiles.

**Comment by ppot** [comment]
@matheuss

**Comment by matheuss** [comment]
I'm not sure if this was fixed on #740 – could you confirm @BenoitAverty?

**Comment by iovis9** [comment]
For me it's working fine till I spawn a tmux session, then it does the opposite: it shifts text left.
<img width="736" alt="screen shot 2016-12-29 at 11 25 21" src="https://cloud.githubusercontent.com/assets/6660202/21541786/886587f2-cdb9-11e6-9df9-621fa4d7d1bd.png">
